templates/package/rollup.config.js

```diff
@@ -0,0 +1,24 @@
+import copy from "rollup-plugin-copy";
+
+export default {
+	input: "src/index.js",
+	output: [
+		{
+			file: "dist/cjs/index.cjs",
+			format: "cjs",
+		},
+		{
+			file: "dist/esm/index.js",
+			format: "esm",
+			banner: '// @ts-self-types="./index.d.ts"',
+		},
+	],
+	plugins: [
+		copy({
+			targets: [
+				{ src: "src/types.ts", dest: "dist/cjs", rename: "types.cts" },
```

Comment:
If the types file will be named `types.cts`, then we should probably add a `build-cts.js` script to update references as in https://github.com/eslint/rewrite/commit/f77ba177d4e4c5d2ed828cfd9a5149df2ccb3a7f.

Reply:
Good point. I can't wait until we only support Node.js 22 and can get rid of CommonJS altogether.